Cargo.toml

```diff
@@ -16,11 +16,13 @@ events = { version = "^0.1.0", path = "packages/events" }
 events-derive = { version = "^0.1.0", path = "packages/events-derive" }
 axelar-wasm-std = { version = "^0.1.0", path = "packages/axelar-wasm-std" }
 axelar-wasm-std-derive = { version = "^0.1.0", path = "packages/axelar-wasm-std-derive" }
+h2 = "0.3.24"
```

Comment:
I'm confused about why we need to add these packages here. I think they are dependencies of our dependencies, which are already specified in this file, so can we just upgrade our dependencies? Like something in this file is already including `h2`, can we just upgrade whatever that is?

Reply:
maybe I misunderstood you, but, having them here should mean that in the per package Cargo.toml files they should change to `{ workspace = true }`

Reply:
as it stands right now I'm also a bit confused why the only change here is to add new dependencies

Reply:
so, what to do instead? The reason we put it here was for all other cargo.tomls to be using it rather than changing the version over multiple files

Reply:
which of our dependencies is using h2? Is it not possible to just update that dependency? h2 is not actually a dependency that we specify in any Cargo.toml file right now, I'd prefer not to add it, but rather just update the dependency that is including it

Reply:
yeah I'm taking this path now.